commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java

```diff
@@ -203,7 +203,9 @@ public CommerceOrder addCommerceOrder(
 
 		validateGuestOrders();
 
-		if (Validator.isNull(commerceCurrencyCode)) {
+		if (Validator.isNull(commerceCurrencyCode) ||
+			Validator.isBlank(commerceCurrencyCode)) {
```

Comment:
This... shouldn't be necessary.
`Validator.isNull` checks for the string being `null`, `""`, or `"null"`.
`Validator.isBlank` checks for the string being `null` or `""`.
So `isNull` is a strict superset of `isBlank`, so we should never have to check both.